ouroboros-consensus/src/Ouroboros/Consensus/Node.hs

```diff
@@ -296,51 +288,37 @@ initNetwork registry nodeArgs kernel RunNetworkArgs{..} = do
       (protocolCodecs (getNodeConfig kernel))
       (protocolHandlers nodeArgs kernel)
 
-    networkAppNodeToNode :: Versions
-                              NodeToNodeVersion
-                              DictVersion
-                              (NetworkApps peer)
-    networkAppNodeToNode =
-      simpleSingletonVersions
-        NodeToNodeV_1
-        (NodeToNodeVersionData { networkMagic = 0 })
-        (DictVersion nodeToNodeCodecCBORTerm)
-        networkApps
-
-    networkAppNodeToClient :: Versions
-                                NodeToClientVersion
-                                DictVersion
-                                (NetworkApps peer)
-    networkAppNodeToClient =
-      simpleSingletonVersions
-        NodeToClientV_1
-        (NodeToClientVersionData { networkMagic = 0 })
-        (DictVersion nodeToClientCodecCBORTerm)
-        networkApps
+    -- TODO: network magics should be configurable, this gives an early fail if
+    -- one tries to run testnet vs mainnet.
```

Comment:
+1 for the comment, we should have a task for it too.

Reply:
Exactly, we need to bug `consensus` team for that :).